auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -0,0 +1,277 @@
+    }
+    ConfigHelper.addToSession(SessionStorageKeys.BusinessIdentifierKey, businessIdentifier)
+    let redirectURL = `${ConfigHelper.getBusinessURL()}${businessIdentifier}`
+
+    window.location.href = decodeURIComponent(redirectURL)
+  }
+
+  /** Emit business/nr information to be unaffiliated. */
+  @Emit()
+  removeBusiness (business: Business): RemoveBusinessPayload {
+    return {
+      orgIdentifier: this.currentOrganization.id,
+      business
+    }
+  }
+
+  /** Apply data table headers dynamically to account for computed properties. */
+  @Watch('selectedColumns', { immediate: true })
+  private applyHeaders (): void {
+    this.headers = [
+      { text: 'Business Name', align: 'start', value: 'name', sortable: true, show: true },
+      { text: 'Number', value: 'number', sortable: false, show: this.showCol('Number') },
+      { text: 'Type', value: 'type', sortable: false, show: this.showCol('Type') },
+      { text: 'Status', value: 'status', sortable: false, show: this.showCol('Status') },
+      { text: 'Folio', value: 'folio', sortable: false, show: this.showCol('Folio') && this.isPremiumAccount },
+      { text: 'Last Modified', value: 'lastModified', sortable: false, show: this.showCol('Last Modified') },
+      { text: 'Modified By', value: 'modifiedBy', sortable: false, show: this.showCol('Modified By') },
+      { text: 'Actions', align: 'end', value: 'action', sortable: false, show: true }
+    ]
+  }
+
+  /** Is true when the selected columns includes the column argument. */
+  private showCol = (col): boolean => {
+    return this.selectedColumns.includes(col)
+  }
+}
+</script>
+<style lang="scss" scoped>
+@import '@/assets/scss/theme.scss';
+
+#affiliated-entity-section {
+  .table-header {
+    display: flex;
+    background-color: $app-lt-blue;
+    padding: .875rem;
+  }
+
+  tbody {
+    tr {
+      vertical-align: top;
+
+      &:hover {
+        cursor: pointer;
+        background-color: $app-lt-blue !important;
+      }
+
+      td {
+        padding: 1rem .875rem;
+        line-height: 1.125rem;
+      }
+
+      td:not(:first-child):not(:last-child) {
+        max-width: 5rem;
+      }
+    }
+  }
+
+  .action-cell {
+    text-align: end !important;
+    padding-right: 0;
+  }
+
+  .actions {
+    .open-action {
+      border-right: 1px solid $gray1;
+    }
+
+    .open-action-btn {
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
+    }
+
+    .more-actions-btn {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+
+    .v-btn + .v-btn {
+      margin-left: 0.5rem;
+    }
+  }
+}
+
+// Vuetify Overrides
+::v-deep .v-list-item {
+  min-height: 2rem !important;
+
+  :hover {
+    cursor: pointer;
+  }
+}
+
+::v-deep .theme--light.v-data-table thead tr:last-child th:last-child span {
+  padding-right: 85px;
+}
+
+::v-deep .theme--light.v-list-item .v-list-item__action-text, .theme--light.v-list-item .v-list-item__subtitle {
+  color: #1669BB;
```

Comment:
Are there CSS variables that could be used here and on line 274?

Reply:
There isn't one that exists yet, but i should add that into our `app` colours. A primary equivalent for outside the templates, will update!

Reply:
Awesome, thanks!